velox/connectors/hive/storage_adapters/hdfs/tests/HdfsFileSystemTest.cpp

```diff
@@ -99,6 +102,14 @@ void readData(ReadFile* readFile) {
   ASSERT_EQ(warfFromBuf, "abbbbbcc");
 }
 
+std::unique_ptr<WriteFile> openFileForWrite(std::string_view path) {
+  auto memConfig = std::make_shared<const core::MemConfig>(configurationValues);
+  std::string hdfsFilePath =
+      "hdfs://" + localhost + ":" + hdfsPort + std::string(path);
```

Comment:
NYC: s/localhost/kLocalHost/ for consts. The same for hdfsPort?

Reply:
the localhost and hdfsPort are variable already, do you means modify the name localhost to kLocalHost, hdfsPort to kHdfsPort?